.github/workflows/kiali-ci.yml

```diff
@@ -0,0 +1,29 @@
+name: Kiali Go CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-20.04
+    steps:
+
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.14
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Build
+      run: make build 
+
+    - name: Test
+      run: make test
```

Comment:
This is very simplified CI.

I guess we have the following in travis:

```
 - make swagger-install
 - make lint-install

script:
  - if [ ! -z "$(gofmt -l ${GO_FILES})" ]; then echo "These files need to be formatted:" "$(gofmt -l ${GO_FILES})";echo "Diff files:"; gofmt -d ${GO_FILES}; exit 1; fi # Gofmt Linter
  - make lint
  - make swagger-travis
  - make clean build test-race
```

Perhaps the goformatted may be packaged as an aux make target, but probably the others steps can be added in the CI in a similar was as we did for travis.